app/gui2/src/assets/main.css

```diff
@@ -18,8 +18,9 @@ body {
 
 #app {
   position: absolute;
+  top: var(--top-bar-height, 0);
   width: 100%;
-  height: 100%;
+  height: calc(100% - var(--top-bar-height, 0));
   display: flex;
```

Comment:
While this is probably fine for now, I'd rather see the #app being positioned and sized naturally through flexbox layout. It would then naturally adapt to any panels created around it, including the top bar.